pipeline/talend_test/harvest_results/6-nec-hob.emii.org.au/soop_ba/soop_ba/soop_ba_trajectory_map.csv

```diff
@@ -0,0 +1,3 @@
+file_id,url,size,time_coverage_start,time_coverage_end,colour,geom,platform_code,vessel_name,freq_18khz_b,freq_38khz_b,freq_70khz_b,freq_120khz_b
+1,IMOS/SOOP/SOOP-BA/E5WW_Will-Watch/Will-Watch_20140604-20140607/IMOS_SOOP-BA_A_20140617T134657Z_E5WW_FV02_Will-Watch-ES60-38_END-20140620T100455Z_C-20150625T053046Z.nc,8614518,2014-06-04 17:28:30,2014-06-07 01:47:24,#5C710B,0102000020E6100000180000008566F05AB6D648403BE4C17174D43BC0537EADA6460F4940B1FBD0FD83933BC020D655604C4749405120606BBE233BC0755AB7416D644940CB9F2DEFC3013BC01EB3C3CAAE9B49408575671636913AC032E6AE25E49B494052D94D7CF7853AC08C23628C40AC494047C0117218633AC0624148B305B94940CA0F85643B493AC0F346BD120EBA49400A230736F8473AC0AFD5BBCB87BA4940CFB6A2E60E4A3AC0C448480718D54940199EEAD2B7173AC0D4A598E64A084A40C4BF5B625DAB39C0B362551045144A40164E1419049A39C0B3987552A1214A409A4CCBFE2F7E39C07A2B7DAE57224A40EA4141295A7C39C0D30228047A214A404E215F5BA27839C0DAFBF95B27984A40CAC3DF82AE8538C01751A8E1BA514B402BBE041B950437C0F39E982F0E534B40B4ABF31C4FF736C03A73B44FEC984B4015002536066936C0F72DE710C09D4B40354AF2B7985E36C078BF027F159F4B404DC59D3D5D5736C0F3FFEC3F19314C4086A1DD3A622A35C07F03ACB8655D4C40043C69E1B2BC34C0,E5WW,Will Watch,f,t,f,f
+2,IMOS/SOOP/SOOP-BA/E5WW_Will-Watch/Will-Watch_20140617-20140620/IMOS_SOOP-BA_A_20140617T134657Z_E5WW_FV02_Will-Watch-ES60-38_END-20140620T100455Z_C-20150625T053046Z.nc,11474635,2014-06-17 13:46:56.999,2014-06-20 10:04:54.999,#5C710B,0102000020E61000000A000000AF4BA66D708F4C401BE31635907034C046527F7BA6154C40B0979E97275E35C087AB3DCB77154B403F954F66EE4737C035ED418C80374A40CBB8EB9DB8EA38C0CE13B63CE26349402DA17197B3743AC06BAFCB11AE5F48403EE346BA34C63BC0BC36028E906F4740807288761DF93CC06EF8BC6662654740F1E2F547B50E3DC0FF8B0C175E60474015CB0C9F9D113DC0CF4013FE98504740D9C9856F06263DC0,E5WW,Will Watch,f,t,f,f
```

Comment:
Looks like we aren't doing an update here.  Looks like the dest path is formed using info from the file so we probably need to update the existing file in a way that causes the file to be updated and so we can see that the update has worked e.g. modifying one of the attributes sourced from the file excluding time coverage.

Reply:
the only way to make sure you have done an update is by checking the `last_indexed `column in table` indexed_file`
to do the update for ex, change the global attribute `data_centre` like this 
`ncatted -h -a data_centre,global,o,c,'AODN' IMOS_SOOP-BA_A_20140617T134657Z_E5WW_FV02_Will-Watch-ES60-38_END-20140620T100455Z_C-20150625T053046Z.nc`